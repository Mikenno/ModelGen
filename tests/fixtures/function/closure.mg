
func add(x)
	return func(y) return x + y

add1 = add(1)
add5 = add(5)

print(add1(3))
print(add5(3))

add1.x = 10
add5.x = 50

print(add1(3))
print(add5(3))
