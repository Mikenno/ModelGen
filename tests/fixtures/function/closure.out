4
8
13
53
